gateway/hail.nginx.conf.in

```diff
@@ -197,12 +197,8 @@ server {
     }
 
     location @pod_dead {
-        return 307 $scheme://$http_host/new;
-    }
-
-    location ~ /instance-ready/([^/]+)/ {
```

Comment:
`/instance-ready/<uuid>` was called only by the websocket, from the notebook-leader server. I removed this endpoint in favor of calling `/instance/<uuid>` directly because `/instance-ready` bypasses auth, and because reaching through the full path is probably a better test of reachability.

Reply:
Doing this is slightly messier though, and requires the websocket calls timeout to be smaller than the proxy_connect_timeout.

Reply:
I reverted this actually; causes refreshes on each call